src/coreclr/tools/aot/ILLink.Shared/DiagnosticId.cs

```diff
@@ -0,0 +1,62 @@
+﻿namespace ILLink.Shared
```

Comment:
These will need copyright headers. We should probably do those changes in the linker repo and then propagate here. Ideally, files in ILLink.Shared will be just straight copies out of the linker repo with no additions.